File: Python/tests/test_banditNarm_lapse_decay.py
import pytest

from hbayesdm.models import banditNarm_lapse_decay


def test_banditNarm_lapse_decay():
    _ = banditNarm_lapse_decay(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_banditNarm_lapse_decay.R
context("Test banditNarm_lapse_decay")
library(hBayesDM)

test_that("Test banditNarm_lapse_decay", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm_lapse_decay(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
